src/test/integration/zerostate/zerostate_test.js

```diff
@@ -18,7 +18,7 @@ describe('Zero state view', () => {
   let page;
 
   beforeEach(() => {
-    browser.get('/index.html');
+    browser.get('#/zerostate');
```

Comment:
Fix of test. It passed on travis in old form, but if there were any replica sets existing then it would redirect to `replicasetlist` page instead of `zerostate`.